emoji/src/main/java/com/vanniktech/emoji/EmojiImageView.java

```diff
@@ -88,7 +88,7 @@ public EmojiImageView(final Context context, final AttributeSet attrs) {
     }
   }
 
-  public void setEmoji(@NonNull final Emoji emoji) {
+  void setEmoji(@NonNull final Emoji emoji) {
```

Comment:
should this be public? otherwise you can't access it when you want to do your custom implementation